src/components/SignInForm.js

```diff
@@ -0,0 +1,116 @@
+import React from "react";
+import styled from "styled-components";
+import { Formik } from "formik";
+import * as Yup from "yup";
+
+const validationSchema = Yup.object({
+  email: Yup.string().email("Invalid email format").required("Required"),
+  password: Yup.string()
+    .min(8, "password must be 8 digit")
+    .required("Required"),
+});
+const onSubmit = (values) => {
+  console.log("form data:", values);
+};
+const initialValues = {
+  email: "",
+  password: "",
+};
```

Comment:
It's better practice to keep ```onSubmit``` & ```initialValues``` inside component because sometimes we need component props values in side both ```onSubmit``` & ```initialValuse```

Reply:
okay